CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 ### Added
 
 - Add support for .NET 7.
+- Add support for `OTEL_DOTNET_AUTO_TRACES_ENABLED`.
```

Comment:
How about adding info in `### Fixed` that the ``OTEL_DOTNET_AUTO_*_ENABLED` are also respected by bytecode instrumentations?